src/vs/workbench/services/actions/common/menusExtensionPoint.ts

```diff
@@ -308,6 +308,12 @@ const apiMenus: IAPIMenu[] = [
 		description: localize('merge.toolbar', "The prominent button in an editor, overlays its content"),
 		proposed: 'contribEditorContentMenu'
 	},
+	{
+		key: 'editor/lineNumber/context',
+		id: MenuId.EditorLineNumberContext,
+		description: localize('editorLineNumberContext', "The contributed editor line number context menu"),
+		proposed: 'contribShareMenu'
```

Comment:
Should this be a dedicated proposal or is the re-use of shareMenu intentional?

Reply:
It's intentional for now because the only extension-contributed actions I foresee coming into this menu are Copy GitHub Permalink and Copy vscode.dev Link initially--but it will probably make sense to make this a standalone proposal soon.